src/Character.js

```diff
@@ -55,25 +50,10 @@ export default class Character extends React.Component<CharacterProps, {}> {
     render() {
         return (
             <g
-                ref={this.characterRef}
```

Comment:
As discussed in our pairing session, we should work to restore the use of refs instead of IDs if possible, as we want to have multiple characters eventually, and won't have a single ID in that case.